server/README.md

```diff
@@ -78,7 +83,7 @@ scratchy-server
 
 * Restart mongodb using docker-compose:
 ```sh
-<scratchy/docker>$ docker-compose up -d
+<scratchy/docker>$ sudo docker-compose up -d
```

Comment:
sudo isn't really needed it depends on the user's settings, this should be addressed with some kind of note ( > blablabla ) saying that sudo should be used if it fails without.
(I only write this on this line but this holds for every command with sudo)

Reply:
For example i didn't need to use sudo on my computer

Reply:
fixes